cmd/build.go

```diff
@@ -91,5 +60,105 @@ func Build(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringVarP(&options.OutputMode, "progress", "", "tty", "show plain/tty build output")
 	cmd.Flags().StringArrayVar(&options.BuildArgs, "build-arg", nil, "set build-time variables")
 	cmd.Flags().StringArrayVar(&options.Secrets, "secret", nil, "secret files exposed to the build. Format: id=mysecret,src=/local/secret")
+	cmd.Flags().StringVarP(&options.Namespace, "namespace", "", "", "namespace against which the image will be consumed. Default is the one defined at okteto context or okteto manifest")
 	return cmd
 }
+
+func isManifestV2(file string) (model.ManifestBuild, bool) {
+	r, err := strconv.ParseBool(os.Getenv("OKTETO_ENABLE_MANIFEST_V2"))
+	if err != nil || !r {
+		return nil, false
+	}
+	mPath := contextCMD.GetOktetoManifestPath(file)
+	if mPath == "" {
+		return nil, false
+	}
+
+	mBuild, err := model.GetBuildManifest(mPath)
+	if err != nil || mBuild == nil {
+		return nil, false
```

Comment:
Should we at least log the error if exists to our logs, so an `okteto doctor` command would include them? It could help us to troubleshoot issues

Reply:
i dont find a way to add logs without displaying them at the cli, at this point we are checking the manifest, fallback is building with v1... if i add log.Error this gets displayed into the cli and it gets built with v1 or, in the cases i've tested, the error of not finding Dockerfile for v1 is displayed before exit.... 
also, okteto doctor i dont see any logs being added into the zip file :/